docs/appendices/release-notes/5.7.3.rst

```diff
@@ -74,6 +74,11 @@ Fixes
   :ref:`postgres-pg_catalog` schema, in order to be compatible with PostgreSQL
   behaviour.
 
+- Fixed an issue that caused ``WHERE`` clause containing ``NOT`` operator on
+  an array type to incorrectly filter empty arrays, e.g.::
+
+      SELECT * FROM t WHERE a != [1];
+
```

Comment:
On master  `= []`, `= [data]`,  `!= []` works fine/same as PG.

Only  `!= [data]` is buggy. 

Maybe can phrase like:
> ...``NOT`` operator on a non-empty array to incorrectly...

Reply:
Actually, as discussed internally, it's a behavior change and not a bug fix. 

It goes to 5.8 as a breaking(?) change.

Reply:
Thanks will do, also removed bug/backport tags